int32 control_state_value